src/v2/Components/LotTimer.tsx

```diff
@@ -67,12 +77,18 @@ export const getTimerCopy = (time, hasStarted) => {
 
   let copy = ""
   let color = "blue100"
+  console.log(time)
+  console.log(extendedBiddingEndAt)
 
   // Sale has not yet started
   if (!hasStarted) {
     copy = `${parsedDays + 1} Day${
       parsedDays >= 1 ? "s" : ""
     } Until Bidding Starts`
+    // entered extended bidding
+  } else if (extendedBiddingEndAt) {
+    copy = `Extended: ${parsedMinutes}m ${parsedSeconds}s`
```

Comment:
What are `parsedMinutes` and `parsedSeconds` here, will this show the correct time remaining before the extended bidding period elapses ? I think those are referring to the amount of time before regular bidding ends.

Reply:
yeah this is sort of an in between PR that doesn't actually reset the timer. It just adds the extended label for the moment. My plan was to make that a separate PR.

Reply:
Well, I think the part that can update/reset the timer will be the websocket thing which we will add later.

But in the meantime, why only the `Extended: ...` label w/o showing the correct amount of time left? I think this should show the correct amount of time (so when this page is loaded after time has been extended, it is correct).

Reply:
so I got this to work and it resets the timer, also works in spec 🥳 , however it fails a commit prehook run with `/Users/nicholasspinazze/Desktop/workspace/force/src/v2/Components/LotTimer.tsx
  85:22  error  React Hook "useTimer" is called in function "getTimerCopy" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter  r` so curious to hear your perspective on how to go about addressing that? Maybe extract this portion out of getTimerCopy somehow?

Reply:
Yes, that should be passed in most likely, or use a different method for extended label. `getTimerCopy` is being called every second (due to the `useTimer` hook usage in https://github.com/artsy/force/pull/9976/files#diff-6fe2759f19ace0536543b1fcda098a5cf71e1970db406a8e118e8ab3cfa32120R20).

So it doesn't make a lot of sense to call `useTimer` _within_ `getTimerCopy`, since it's not itself a hook or component.

Reply:
Ohhh I see! That's clever! I was so busy fiddling within the logic of getTimer copy I hadn't even considered that!